approvals/README.md

```diff
@@ -1,10 +1,18 @@
 # Sym Approvals Quickstart
 
-This is a starter template to get a Sym Approval Flow set up for your team.
+This is a starter template similar to the Terraform generated by `symflow init`, but with some variable refactoring that an implementer might consider
+before deploying their configuration to Production.
 
-## Tutorial
+Several variables have been pulled out into a `teraform.tfvars` file. Notably:
+- `environment_name` and `error_channel_name`.
 
-Check out a step-by-step tutorial [here](https://docs.symops.com/docs/deploy-sym-platform).
+In this example, `main` has been pulled out into a variable `environment_name`. This allows this `main.tf` to be applied
+with a different `terraform.tfvars`, which will then create a different set of resources (e.g. `sandbox` vs `main`)!
+
+This is helpful if you wish to test your configuration with resources in a test environment before configuring Sym with Production data.
+
+Furthermore, `impl.py` has been modified with an `on_approve` and `on_deny` hook that only allows a subset of users defined in `terraform.tfvars > flow_variables["appprovers"]`
```

Comment:
I think "furthermore" is overly formal here -- let's just go with "In addition."

Also,
> hook that only allows a subset→
"hook that allows only a subset"